samples/project-anfield/import_cluster_config.py

```diff
@@ -745,6 +828,12 @@ def create_protection_jobs():
                                 "following protocol %s. Supported protocol is kNfs." % (
                                     job_name, name, ", ".join(protocol)))
                             break
+                if environment == "kCassandra":
```

Comment:
make a constant KCASSANDRA at the top and use that constant in all places.

Reply:
Okay